### PR TITLE
[14.0][MIG]stock_change_qty_reason: Migration to 14.0

### PR DESCRIPTION
**Pull request by GuillemCForgeFlow**
Standard migration
@ForgeFlow

### CODE REVIEW

**Review by JoanMForgeFlow — COMMENTED** [review]
LGTM! thanks

**Review by JoanMForgeFlow — APPROVED** [review]

**Review by JoanMForgeFlow — APPROVED** [review]
LGTM! thanks

**Review by MiquelRForgeFlow — APPROVED** [review]

File: stock_change_qty_reason/views/stock_inventory_line_view.xml [hunk 1]
```diff
@@ -24,28 +24,4 @@
             </field>
         </field>
     </record>
-    <record id="reason_stock_inventory_line_tree2" model="ir.ui.view">
-        <field name="name">stock.inventory.line.tree</field>
-        <field name="model">stock.inventory.line</field>
-        <field name="inherit_id" ref="stock.stock_inventory_line_tree2" />
-        <field name="arch" type="xml">
-            <field name="location_id" position="before">
-                <field name="reason" />
```
Comment (sebalix) [inline stock_change_qty_reason/views/stock_inventory_line_view.xml]: What has motivated this change? This view doesn't exist anymore on v14?

**Review by sebalix — COMMENTED** [review]
Can you squash together `[UPD] Update stock_change_qty_reason.pot` commits, as well as for `Update translation files` ones? This is to reduce the history of commits.

File: stock_change_qty_reason/views/stock_inventory_line_view.xml (hunk 1 above)
Comment (GuillemCForgeFlow) [reply stock_change_qty_reason/views/stock_inventory_line_view.xml]: Exactly, the two views that end with tree2, they don't exist anymore on version 14. So, as everything now is displayed on the other two views, I deleted the other two.

**Comment by GuillemCForgeFlow** [comment]
> Can you squash together `[UPD] Update stock_change_qty_reason.pot` commits, as well as for `Update translation files` ones? This is to reduce the history of commits.

Hope that works! @sebalix

**Comment by sebalix** [comment]
@GuillemCForgeFlow Hum I think you removed the commits and squashed them somewhere in another commits, you should keep one of them instead :) Example:
```
- commit1
- [UPD] Update stock_change_qty_reason.pot
- commit2
- Update translation files
- [UPD] Update stock_change_qty_reason.pot
- commit3
- Update translation files
```
Has to become:
```
- commit1
- [UPD] Update stock_change_qty_reason.pot
- commit2
- Update translation files
- commit3
```
The last administrative commits were merged into the first ones (see https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0#tasks-to-do-in-the-migration)

File: stock_change_qty_reason/views/stock_inventory_line_view.xml (hunk 1 above)
Comment (sebalix) [reply stock_change_qty_reason/views/stock_inventory_line_view.xml]: thank you!

**Comment by GuillemCForgeFlow** [comment]
> @GuillemCForgeFlow Hum I think you removed the commits and squashed them somewhere in another commits, you should keep one of them instead :) Example:
> 
> ```
> - commit1
> - [UPD] Update stock_change_qty_reason.pot
> - commit2
> - Update translation files
> - [UPD] Update stock_change_qty_reason.pot
> - commit3
> - Update translation files
> ```
> 
> Has to become:
> 
> ```
> - commit1
> - [UPD] Update stock_change_qty_reason.pot
> - commit2
> - Update translation files
> - commit3
> ```
> 
> The last administrative commits were merged into the first ones (see https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0#tasks-to-do-in-the-migration)

Hope now I didn't get it wrong. I had to squash all the `[UPD] Update stock_change_qty_reason.pot` and `Update translation files` commits with their respective previous. Because If I tried changing the history I had conflicts with the *.po files.
Thanks for the help!
@sebalix

**Comment by rousseldenis** [comment]
@GuillemCForgeFlow Seems great to me

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**